sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx

```diff
@@ -29,11 +74,74 @@ const NetworksSwitcher: FC<NetworksSwitcherProps> = () => {
 		}
 	};
 
+	const OPTIMISM_OPTIONS = [
+		{ label: 'header.networks-switcher.l1', postfixIcon: 'Switch', onClick: switchToL1 },
+		{
+			label: 'header.networks-switcher.optimistic-gateway',
+			postfixIcon: 'Link',
+			link: 'https://gateway.optimism.io/',
+		},
+		{
+			label: 'header.networks-switcher.optimistic-etherscan',
+			postfixIcon: 'Link',
+			link: blockExplorerInstance?.baseLink,
+		},
+		{
+			label: 'header.networks-switcher.learn-more',
+			postfixIcon: 'Link',
+			link: 'https://blog.kwenta.io/hello-optimism-kwenta-is-live-on-l2/',
+		},
+	];
+
+	const formatOptionLabel = ({
+		label,
+		prefixIcon,
+		postfixIcon,
+		link,
+		onClick,
+	}: ReactSelectOptionProps) => (
+		<ExternalLink href={link} onClick={onClick}>
+			<LabelContainer noPadding={!!prefixIcon}>
+				{prefixIcon === 'Optimism' && <PrefixIcon src={OptimismIcon} height={12} />}
+				{t(label)}
+				{postfixIcon &&
+					(postfixIcon === 'Link' ? <Svg src={LinkIcon} /> : <Svg src={SwitchIcon} />)}
+			</LabelContainer>
+		</ExternalLink>
+	);
+
+	const DropdownIndicator = (props: any) => {
+		return (
+			<components.DropdownIndicator {...props}>
+				<StyledCaretDownIcon
+					src={CaretDownIcon}
+					viewBox={`0 0 ${CaretDownIcon.width} ${CaretDownIcon.height}`}
+				/>
+			</components.DropdownIndicator>
+		);
+	};
+
 	return !isL2 ? (
 		<Container onClick={switchToL2}>
-			<Button>{t('header.networks-switcher.l2')}</Button>
+			{!isMainnet && <Button>{t('header.networks-switcher.l2')}</Button>}
```

Comment:
would we rather just not render the container at all? if we're not on mainnet

Reply:
yeah that's better but im going to revert this in a week